LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoRaPayloadDataLns.cs

```diff
@@ -51,20 +52,18 @@ MacMessageType.JoinAccept or
             _ = macHeader.Write(Mhdr.Span);
 
             // Setting Fctrl
-            Fctrl = new byte[1];
-            _ = control.Write(Fctrl.Span);
+            FrameControlFlags = fctrlFlags;
 
             // Setting Fcnt
             Fcnt = new byte[sizeof(ushort)];
             BinaryPrimitives.WriteUInt16LittleEndian(Fcnt.Span, counter);
 
             // Setting FOpts
-            var foptsSize = control.OptionsLength;
-            Fopts = new byte[foptsSize];
+            Fopts = new byte[options.Length / 2];
```

Comment:
When I checked I did not see this logic asserted in unit tests. I can't see how the division by 2 comes in, could you explain?

Reply:
`options` is a hexdecimal-encoded string therefore the byte length will be half of the string length.

Reply:
Created issue #1014 to track the lack of tests for `LoRaPayloadDataLns`.